alembic/versions/a4a79802ba51_allow_api_and_print_jobs_to_not_have_.py

```diff
@@ -11,7 +11,7 @@
 revision = 'a4a79802ba51'
 down_revision = 'a5d2a3700b1a'
 branch_labels = None
-depends_on = None
+depends_on = 'c7a439f29c1c'
```

Comment:
This was something I ran into along the way. We appear to have two alembic heads at the moment, and they aren't completely independent. I think this depends is sufficient to clean it up, but we may want to reset our alembic history at some point.

Reply:
Whoops. Yea, maintaining two forks means we end up with two branches of migrations. I try to manually rebase them but I must have messed that up this time.

Actually I wonder if it's worth it to maintain a separate repo for Alembic. It'd be more overhead when making normal changes but I'll usually avoid making major DB changes in a divergent fork, it will just be like "MFF changes for art show and whatever" is one branch and "MAGFest changes for MIVS" is another.